Makefile

```diff
@@ -1,7 +1,7 @@
-KIND_VERSION = 0.7.0
-KUBERNETES_VERSION = 1.15.7
+KIND_VERSION = 0.8.1
+KUBERNETES_VERSION = 1.17.9
 KIND_NODE_HASH = e2df133f80ef633c53c0200114fce2ed5e1f6947477dbc83261a6a921169488d
```

Comment:
@cw-sakamoto 
https://github.com/kubernetes-sigs/kind/releases/tag/v0.8.0
```
kindest/node:v1.18.2@sha256:7b27a6d0f2517ff88ba444025beae41491b016bc6af573ba467b70c5e8e0d85f
```
You will need to change the hash.